### ISSUE TITLE
Get Dist build in working order

### ISSUE THREAD

**Issue by BlakeDraper** [USGS-WiM/STNWeb#107]
Currently the production/distribution build is broken. This has been a low priority in relation to getting things working, but should get done for initial pre-release.

**Comment by BlakeDraper** [comment]
UPDATE: This is almost working, and I know exactly the problem, just not yet the solution.
The gulpfile is for some reason not injecting the vendor.min.css to the output dist/index.html. But when I manually added it (before app.min.css and stn.min.css of course), the app comes out fine. So after NC trip just need to track down why the gulp task is failing to get the vendor.min.css injected. 

Note: did once manage to get the vendor.min.css to inject when it was listed after the appStyles inject pipe, but then the the app styles were not injected.

``` javascript
pipes.builtIndexProd = function() {

    var vendorScripts = pipes.builtVendorScriptsProd();
    var appScripts = pipes.builtAppScriptsProd();
    var appStyles = pipes.builtAppStylesProd();
    var vendorStyles = pipes.builtVendorStylesProd();

    return pipes.validatedIndex()
        .pipe(gulp.dest(paths.dist)) // write first to get relative path for inject
        .pipe(plugins.inject(vendorScripts, {relative: true, name: 'bower'}))
        .pipe(plugins.inject(appScripts, {relative: true}))
        .pipe(plugins.inject(vendorStyles, {relative: true}))
        .pipe(plugins.inject(appStyles, {relative: true}))
        .pipe(plugins.htmlmin({collapseWhitespace: true, removeComments: true}))
        .pipe(gulp.dest(paths.dist));
};
```

**Comment by BlakeDraper** [comment]
Not sure how, but got this working. I messed with a few things, but shouldn't have affected this. At any rate, we seem to have a working dist build now for deployment. Fixed with PR #208.
